src/Serializer/SerializerContextBuilder.php

```diff
@@ -95,7 +95,7 @@ public function createFromRequest(Request $request, bool $normalization, ?array
 
         if (!$normalization) {
             if (!isset($context['api_allow_update'])) {
-                $context['api_allow_update'] = \in_array($method = $request->getMethod(), ['PUT', 'PATCH'], true);
+                $context['api_allow_update'] = \in_array($method = $request->getMethod(), ['PUT', 'PATCH', 'POST'], true);
```

Comment:
post should not update I think

Reply:
No, it should not. But it should be able to reference an existing relation, like it is done with PUT and PATCH. Not for updating an existing resource, but for creating one. In which case `api_allow_update` is probably a misnomer.

Reply:
You are right, due to my internal tests, I was under the wrong impression that a plain IRI requires this. But it does not. Closing.